### PR TITLE
add email logo

### PR DESCRIPTION
**Pull request by instil-spencerc**
Makes progress on https://github.com/instilus/planning/issues/352

### CODE REVIEW

File: emails/layouts/generic-layout.pug [hunk 1]
```diff
@@ -7,6 +7,7 @@ link(rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&dis
 link(rel="stylesheet", href="styles/shared.css", data-inline)
 
 #wrapper
+  img#logo(src="https://home.dev.instil.us/static/media/emailImageLogo.84fa3ce0.png" alt="Instil Logo" width="118" height="46")
```
Comment (instil-jon) [inline emails/layouts/generic-layout.pug:10]: Can we push this logo to the production domain or does it have to live on dev?

Comment (instil-spencerc) [reply emails/layouts/generic-layout.pug:10]: Good point. I'll take care of that shortly, and update the PR.

Comment (instil-spencerc) [reply emails/layouts/generic-layout.pug]: @instil-jon updated: https://github.com/instilus/email-templates/pull/4/files#diff-f5486517a7c9328fbc3cd1139a39b596836da11df369bf5e230c246992479d3eR10

Comment (instil-spencerc) [reply emails/layouts/generic-layout.pug]: @instil-jon bump

**Review by instil-laura — APPROVED** [review]

**Pull request merged**